frontend/src/app/modules/availability/new-slot/questions/questions.component.html

```diff
@@ -1,41 +1,62 @@
 <div class="container flex-col">
-    <mat-label class="outside-label bold">Questions for Booker</mat-label>
+    <mat-label class="title bold">Required Questions</mat-label>
     <!-- Mandatory questions -->
     <ng-container *ngIf="mandatoryQuestionsExist">
         <ng-container *ngFor="let question of questions; let i = index">
             <ng-container *ngIf="question.isMandatory">
-                <div class="question-mandatory list-item-wrapper flex cross-axis-center">
+                <div class="question-mandatory list-item-wrapper flex cross-axis-center bottom">
                     <mat-label class="question-index bold">{{ i + 1 }}.</mat-label>
-                    <mat-form-field class="question-input no-padding" appearance="outline">
-                        <input matInput type="text" [(ngModel)]="question.questionText" disabled />
+                    <mat-label class="bold">{{ question.questionText }}</mat-label>
+                </div>
+                <div class="question-mandatory list-item-wrapper flex cross-axis-center">
+                    <mat-form-field class="question-input no-padding max-input-width" appearance="outline">
+                        <input matInput type="text" [placeholder]="question.placeHolderText" disabled />
                     </mat-form-field>
+                    <div class="checkbox-container flex cross-axis-center">
+                        <mat-checkbox name="questionCheckBox" checked="true"> </mat-checkbox>
+                        <span class="input-message">{{ checkBoxTexts[i] }}</span>
+                    </div>
                 </div>
```

Comment:
How will we save these checkboxes in the future with the current implementation?

Reply:
We need to exclude this checkboxes from ng container and make them as stand alone elements ?

Reply:
I added to component property for saving data from check boxes. We will work with them in future (when add logic for this check boxes values)

Reply:
I didn't see logic for saving checkboxes

Reply:
And such implementation looks strange and not reusable